cypress/integration/cucumber/step_definitions/login-test.js

```diff
@@ -0,0 +1,37 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+/// <reference types="Cypress" />
+
+beforeEach(() => {
+  cy.initiateExceptionsLogger();
+});
+
+afterEach(() => {
+  cy.end();
+});
+
+Given('User opens the website at {string}', (url) => {
```

Comment:
you would only need a parameter if your feature file expects one. checkout how we use `scenario outline`, this review applies for everywhere you have used parameters